app/src/main/java/org/sopt/pingle/presentation/ui/onboarding/OnboardingExplanationAdapter.kt

```diff
@@ -0,0 +1,65 @@
+package org.sopt.pingle.presentation.ui.onboarding
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.ListAdapter
+import androidx.recyclerview.widget.RecyclerView
+import org.sopt.pingle.databinding.ItemOnboardingExplanationDescriptionBinding
+import org.sopt.pingle.databinding.ItemOnboardingExplanationTitleBinding
+import org.sopt.pingle.presentation.type.OnboardingExplanationType
+import org.sopt.pingle.util.view.ItemDiffCallback
+
+class OnboardingExplanationAdapter(context: Context) :
+    ListAdapter<String, RecyclerView.ViewHolder>(
+        ItemDiffCallback<String>(
+            onContentsTheSame = { old, new -> old == new },
+            onItemsTheSame = { old, new -> old == new }
+        )
+    ) {
+    private val inflater by lazy { LayoutInflater.from(context) }
+    private var onBoardingExplanationList = OnboardingExplanationType.values()
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): RecyclerView.ViewHolder {
+        return when (viewType) {
+            TITLE_VIEW_TYPE -> OnboardingExplanationTitleViewHolder(
+                ItemOnboardingExplanationTitleBinding.inflate(inflater, parent, false)
+            )
+
+            DESCRIPTION_VIEW_TYPE -> OnboardingExplanationDescriptionViewHolder(
+                ItemOnboardingExplanationDescriptionBinding.inflate(inflater, parent, false)
+            )
+
+            else -> throw IllegalArgumentException("$VIEWTYPE_EXCEPTION_MESSAGE $viewType")
+        }
+    }
+
+    override fun onBindViewHolder(holder: RecyclerView.ViewHolder, position: Int) {
+        when (holder) {
+            is OnboardingExplanationTitleViewHolder -> holder.onBind()
+            is OnboardingExplanationDescriptionViewHolder -> {
+                var decPosition = position - POSITION_MINUS
+                holder.onBind(onBoardingExplanationList[decPosition])
```

Comment:
```suggestion
                holder.onBind(onBoardingExplanationList[position - POSITION_MINUS])
```
굳이 변수화를 하지 않아도 될 것 같다는 생각입니다